src/answearDisplay/answearDisplay.js

```diff
@@ -0,0 +1,34 @@
+// Creating answearDisplay class
+
+class answearDisplay {
+    constructor(answear, element){
+        this.answear = answear;
+        this.element = element;
```

Comment:
in my opinion, better way to reach the goal will be pass id of correct element as an argument (instead of whole html element) to your class and then set some styles

Reply:
I changed it to:
this.element = document.getElementById(elementId);
Which should work.